### PR TITLE
feat: add dart tasks

### PR DESCRIPTION
**Pull request by shan-shaji**
**What kind of change does this PR introduce?**
Feature

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
N/A

**Summary**
This PR is still a work in progress and it introduces beginners tasks to learn dart language.

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: src/commands/submit.js [hunk 1]
```diff
@@ -288,6 +288,18 @@ const submitTask = async () => {
...
+      if (err) {
+        logger.error(' Oops there is something wrong with the syntax part!');
+        process.exit(1);
+      }
+      exec(`dart run ${solutionFile}`, (err, solution) => {
+        if (err) throw err;
+        validateSolution(submittedFile);
+        checkSolution(result, solution, exercises.length);
+      });
+    });
```
Comment (ajomadlabs) [inline src/commands/submit.js]: Guess this if else block can be refactored by a switch case and can combine the general steps into a function and be invoked depending on each case 
cc: @jamesgeorge007

Comment (shan-shaji) [reply src/commands/submit.js]: Hey @ajomadlabs can I work on that?

Comment (ajomadlabs) [reply src/commands/submit.js]: @shan-shaji Yes you can go ahead and make the required changes

**Comment by jamesgeorge007** [comment]
Oops, accidentally closed along with the Dependabot PRs. @shan-shaji, were you considering moving forward with this?
Let me know, and I can re-open :)

**Pull request closed without merging**